pkg/apis/componentconfig/types.go

```diff
@@ -47,6 +47,9 @@ type DeschedulerConfiguration struct {
 	// MaxNoOfPodsToEvictPerNode restricts maximum of pods to be evicted per node.
 	MaxNoOfPodsToEvictPerNode int
 
+	// MaxNoOfPodsToEvictPerNamespace restricts maximum of pods to be evicted per namespace.
+	MaxNoOfPodsToEvictPerNamespace int
```

Comment:
No need to define this field here. It's never set.

Reply:
Fix done~